tests/protocol/test_wire.py

```diff
@@ -1205,123 +1204,6 @@ def test_update_goal_state_should_not_persist_the_protected_settings(self):
                     "Could not find the expected number of redacted settings. Expected {0}.\n{1}".format(len(protected_settings), extensions_config))
 
 
-class TryUpdateGoalStateTestCase(HttpRequestPredicates, AgentTestCase):
-    """
-    Tests for WireClient.try_update_goal_state()
-    """
-    def test_it_should_return_true_on_success(self):
-        with mock_wire_protocol(mockwiredata.DATA_FILE) as protocol:
-            self.assertTrue(protocol.client.try_update_goal_state(), "try_update_goal_state should have succeeded")
-
-    def test_incomplete_gs_should_fail(self):
```

Comment:
This test was actually moved to the new `test_goal_state.py`, correct?

Reply:
oh, yes, only this one

Reply:
(not sure why it was with the tests for try_update_goal_state; maybe because there was not a test class for goal_state.py)

Reply:
I think I wrote this test, and I'm fairly certain that was the reason it was there, lol